directory_forms_api_client/actions.py

```diff
@@ -32,19 +34,14 @@ def __init__(
         }
         super().__init__(*args, **kwargs)
 
-    def serialize_data(self, data):
-        return {
-            'data': data,
-            'meta': self.meta
-        }
-
 
 class ZendeskAction(AbstractAction):
 
-    def serialize_data(self, data):
-        return {
-            'data': data,
-            'meta': {
-                'action_name': 'zendesk',
-            }
+    def __init__(self, subject, full_name, email_address, *args, **kwargs):
+        self.meta = {
```

Comment:
like in https://github.com/uktrade/directory-forms-api/pull/6, we need to namespace tickets with the app - guess that can be resolved in api based on the key used

Reply:
to be nice and explicit I've added `DIRECTORY_FORMS_API_NAMESPACE`